confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -59,32 +60,30 @@ def __init__(self, url, max_schemas_per_subject=1000):
         # subj => { schema => version }
         self.subject_to_schema_versions = defaultdict(dict)
 
-    def _send_request(self, url, method='GET', body=None, headers=None):
-        if body:
-            body = json.dumps(body)
-            body = body.encode('utf8')
-        _headers = dict()
-        _headers["Accept"] = ACCEPT_HDR
+        self._session = requests.Session()
+        with self._session as s:
+            if ca_location:
+                s.verify = ca_location
+            if (cert_location or key_location) is not None:
+                if (key_location and cert_location) is None:
+                    raise ValueError(
+                        "Both schema.registry.ssl.certificate.location and schema.registry.ssl.key.location"
+                        "must be set: {} {}".format(cert_location, key_location))
```

Comment:
I don't think there is much value in printing the values

Reply:
Perhaps but it helps when evaluating failures through a third-party. Would you be more okay with returning just the property that is missing?

Reply:
There is no indication in this format what the output is after the :, which becomes even more confusing if empty strings are passed. I don't think the values themselves add much information anyway, the error message as in it is crystal clear.